internal/monitors/repotester/repotester.go

```diff
@@ -0,0 +1,86 @@
+package repotester
+
+import (
+	"context"
+
+	argoapi "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
+	"github.com/argoproj/argo-cd/v2/util/git"
+	api "github.com/flightctl/flightctl/api/v1alpha1"
+	"github.com/flightctl/flightctl/internal/service"
+	"github.com/flightctl/flightctl/internal/store"
+	"github.com/flightctl/flightctl/internal/util"
+	"github.com/flightctl/flightctl/pkg/log"
+	"github.com/flightctl/flightctl/pkg/reqid"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type API interface {
+	Test()
+}
+
+type RepoTester struct {
+	log       logrus.FieldLogger
+	db        *gorm.DB
+	repoStore service.RepositoryStoreInterface
+}
+
+func NewRepoTester(log logrus.FieldLogger, db *gorm.DB, store *store.Store) *RepoTester {
+	return &RepoTester{
+		log:       log,
+		db:        db,
+		repoStore: store.GetRepositoryStore(),
+	}
+}
+
+func (r *RepoTester) Test() {
```

Comment:
where is it used? 
The name is a bit confusing because the standard for golang tests is a function starting with Test, but not a big deal

Reply:
I will change it to TestRepo.  This is the method that the thread calls - set in cmd/flightctl-server/main.go